modules/virt-defining-apps-for-dr.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-disaster-recovery.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-defining-apps-for-dr{context}"]
+= Defining applications for disaster recovery
+
+Define applications for disaster recovery by using either {rh-rhacm-first}-managed VMs or RHACM-discovered VMs.
+
+An RHACM-managed application that includes a VM must be created by using a GitOps workflow and by creating an RHACM Application or `ApplicationSet`.
+
+You can configure any other VM in the cluster as an RHACM-discovered application. This includes VMs imported by using the Migration Toolkit for Virtualization (MTV), VMs created by using the {VirtProductName} web console, or VMs created by any other means, such as the CLI. Applications in an RHACM-discovered VM are discovered by the VM through user-defined labels.
+
+[id="best-practices-rhacm-managed-vm_{context}"]
+== Best practices when defining an RHACM-managed VM
+
+There are several actions you can take to improve your experience and chance of success when defining an RHACM-managed VM.
+
+[discrete]
+[id="use-a-pvc-and-populator_{context}"]
+=== Use a PVC and populator to define storage for the VM
+Because data volumes create persistent volume claims (PVCs) implicitly, data volumes and VMs with data volume templates do not fit as neatly into the GitOps model.
+
+[discrete]
+[id="use-import-method_{context}"]
+=== Use the import method when choosing a population source for your VM disk
```

Comment:
It may make sense to mention that using import works around limitations in Regional DR that prevent you protecting VMs that use cloned PVCs.